src/api/migrations/20190620192200-create-user.js

```diff
@@ -0,0 +1,70 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Users', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    username: {
+      type: Sequelize.STRING
+    },
+    email: {
+      type: Sequelize.STRING
+    },
+    password: {
+      type: Sequelize.STRING
+    },
+    country: {
```

Comment:
@nkpremices are country,  bio, address, gender, profileImage and phoneNumber required fields? If they aren't, then we should allow them to be null.

Also, what do the fields `facebook` and `twitter` represent?

Reply:
Actually ``sequelize``` allows null by default. 
@alainburindi told me that they will be the links to social media profiles

Reply:
Thanks @nkpremices